src/EnergyPlus/DXCoils.cc

```diff
@@ -15329,11 +15383,40 @@ void CalcTwoSpeedDXCoilStandardRating(EnergyPlusData &state, int const DXCoilNum
                 EIR = 0.0;
             }
 
-            TotCapFlowModFac = CurveValue(state, state.dataDXCoils->DXCoil(DXCoilNum).CCapFTemp2, AirMassFlowRatio);
+            // TotCapFlowModFac = CurveValue(state, state.dataDXCoils->DXCoil(DXCoilNum).CCapFFlow(1), AirMassFlowRatio);
             TotCapTempModFac = CurveValue(state,
```

Comment:
This is the major fix part for this problem---the final solution is actually to just remove this line of code, as it was already done once a couple of lines above and the value has not been changed since.

What's showing also showing here in the commented line (15386) is the first  trial to fix by just correct the curve name. However, it is later confirmed that this entire line of code could be complete removed as it is redundant.

Reply:
Interesting fix.  I wasn't expecting the major fix to be removing lines of code, but I basically see what was going on here.